### DISCUSSION TITLE
Users would like to limit ability to update trees via Workbench but retain ability to do it via forms

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5518] [Feature Requests]
UT Austin paleo has requested to be able to prevent students from being able to update tree data via the WorkBench but still want to be able to do it via the forms.

**Comment by maxpatiiuk** [comment]
Fixed in 7.6.00
Users can set tree tables as "Match Only" in the Upload Plan.
Also, the upload plan can be reused between the Data Sets, with "Match Only" settings persisting

**Comment by benanhalt** [comment]
There is nothing stopping whoever is using the workbench from turning off those options. So not sure this entire meets the use case.

**Comment by maxpatiiuk** [comment]
How about an option to turn on "Must Match" be default for certain tables?
They can still go and manually turn it off, but that's outside the original use case - preventing accidental tree updates.

**Comment by benanhalt** [comment]
This is really more part of the permissions system that will be in the future. We have to decide the mechanisms to control who has access to forms and the workbench. I agree the must match feature mitigates the issue, but it doesn't totally resolve it. Let's leave this issue as it is until we tackle permissions.

**Comment by maxpatiiuk** [comment]
This is not covered by the permission system (at least in the current incarnation)
A solution might be do add a collection-level pref that sets trees as must match in the wb mapper

**Comment by grantfitzsimmons** [comment]
> A solution might be do add a collection-level pref that sets trees as must match in the wb mapper

This would need to only apply to non-administrator users as discussed in our meeting yesterday
